packages/babel-traverse/src/path/modification.js

```diff
@@ -126,7 +126,10 @@ export function insertAfter(nodes) {
     if (Array.isArray(this.container)) {
       return this._containerInsertAfter(nodes);
     } else if (this.isStatementOrBlock()) {
-      if (this.node) nodes.unshift(this.node);
+      // Unshift current node if it's not an empty expression
+      if (this.node && (!this.isExpressionStatement() || this.node.expression != null)) {
```

Comment:
I can't find `EmptyExpression` or `isEmptyExpression` in the codebase. Or did you mean we should create a new method `isEmptyExpression` to wrap the above? Like:

```js
if (this.node && !this.isEmptyExpression()) {...}
...
isEmptyExpression() => this.isExpressionStatement() && this.node.expression == null
```